src/masterCompanionMovie.coffee

```diff
@@ -47,12 +57,24 @@ class MasterCompanionMovie
       @companionMovie.triggerEvent("movie:play")
     )
 
+  setupFullScreenMovie = (sourceUrl, wrapperElement, masterCompanionMovie) ->
+    return unless sourceUrl
+    window.fullScreenVideo = new FullScreenVideo(sourceUrl, wrapperElement)
+    masterCompanionMovie.bindEvent("movie:tryFullScreen", ->
+      masterCompanionMovie.triggerEvent("movie:fullscreen")
+      fullScreenVideo.play()
+      masterCompanionMovie.triggerEvent("movie:moveToContent")
+    )
+
   getEventEmitter = () -> new EventEmitter(document.getElementsByTagName("body")[0])
 
   start: () ->
     @startTime = new Date()
     @_timerId = setInterval(=>
       if @trackingsReady
+        mediaUrl = @vastReqest.mediaUrl
+        if mediaUrl && mediaUrl != ""
+          setupFullScreenMovie(, @fullScreenVideoWrapperElement, this)
```

Comment:
[stdin]:77:32: error: unexpected ,
          setupFullScreenMovie(, @fullScreenVideoWrapperElement, this)
                               ^